docs/flatpak-bundle.md

```diff
@@ -0,0 +1,34 @@
+# Anatomy of a Knowledge App Flatpak Bundle
+
+Let's look at a simple knowledge app. Knowledge apps are shipped as part of a bundle, and
+you can fetch a bundle from our Flatpak repository here:
+`flatpak install com.endlessm.astronomy.es eos3`. I chose com.endlessm.astronomy.es for its
```

Comment:
Should we leave some comment about inspecting the resulting flatpak in /var/lib/flatpak?